projects/packages/sync/src/modules/class-posts.php

```diff
@@ -787,8 +806,10 @@ public function send_published( $post_ID, $post ) {
 	 *
 	 * @param array $args The hook parameters.
 	 * @return array $args The expanded hook parameters.
+	 * @deprecated since $$next-version$$
 	 */
 	public function add_term_relationships( $args ) {
+		_deprecated_function( __METHOD__, 'next-version' );
```

Comment:
```suggestion
		_deprecated_function( __METHOD__, '$$next-version$$' );
```

I think this is how we are supposed to use next-version. Btw missed that with https://github.com/Automattic/jetpack/pull/41433 could we fix it real quick?